### PR TITLE
GitHub releases

### PR DESCRIPTION
**Pull request by tracernz**
This is my attempt to add the latest release info to the homepage automatically using the Github API. It's probably a little hacky but that is the extent of my JS ability. :P It looks like this:
Preview
Clicking the show full release notes drops down the whole thing like so:
Preview

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,6 +17,9 @@
 
 <h1> Updates </h1>
 
+<div id="latest-release"></div>
+
+<h2>News</h3>
```
Comment (solidgoldbomb) [inline index.html]: Closing tag should be ```</h2>```.

File: js/releases.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	html += "<a href=\""+json.html_url+"\">";
+	html += json.name;
+	html += "</a>";
+	if(json.prerelease) {
+		html += "<span class=\"rel-testing\">Testing</span>";
+	}
+	else {
+		html += "<span class=\"rel-stable\">Stable</span>";
+	}
+	html += "</h3>";
```
Comment (solidgoldbomb) [inline js/releases.js]: Closing tag should be ```</h2>```.

File: js/releases.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	html += "</ul>";
+
+	$("#latest-release").html(html);
+
+	var relNotesHeightCompact = 150;
+	var relNotesHeightFull = $('#rel-notes').height();
+	$('#rel-notes').height(relNotesHeightCompact);
+
+	$("#toggle-rel-notes").click(function() {
+		if($('#rel-notes').height() == relNotesHeightCompact) {
```
Comment (solidgoldbomb) [inline js/releases.js]: I don't really know JS very well but what happens here if the actual height starts out less than 150?  Will clicking in the div result in it expanding to 150?

**Comment by solidgoldbomb** [comment]
Having the release on the main page like that is a nice feature.  Thanks for the PR.

Could you add a section to the readme that includes pointers to where you got the .js assets?

**Comment by tracernz** [comment]
Thanks for the review @solidgoldbomb ! I appreciate your careful attention in picking up my silly last minute bugs. :+1: I've modified the compact release notes logic so it's only used when needed and tidied up those closing tags.

**Comment by solidgoldbomb** [comment]
Looks good.  Thanks.

**Pull request merged**